### PR TITLE
fixed example

### PR DESCRIPTION
**Pull request by xat**

### CODE REVIEW

File: _posts/en/2017-01-19-binding-objects-to-functions.md [hunk 1]
```diff
@@ -38,19 +38,19 @@ A copy of the given function along with the specified `this` value and initial a
...
 const myCar = {
  brand: 'Ford',
- type: 'Sedan'
- Color: ‘Red’
+ type: 'Sedan',
+ color: 'Red'
 };
 
-const getBrand = () => {
+const getBrand = function() {
```
Comment (loverajoel) [inline _posts/en/2017-01-19-binding-objects-to-functions.md:45]: keep the arrow function syntax

Comment (xat) [reply _posts/en/2017-01-19-binding-objects-to-functions.md:45]: you can't bind arrow functions

Comment (loverajoel) [reply _posts/en/2017-01-19-binding-objects-to-functions.md:45]: true, sorry 👽

**Comment by loverajoel** [comment]
Thanks!

**Pull request merged**